main.go

```diff
@@ -212,9 +221,15 @@ func main() {
 	discover.NewFarcasterDiscover(db, neynarcli).Run(mainCtx, indexer)
 	defer mainCtxCancel()
 
+	// Create Airstack artifact
+	as, err := airstack.NewAirstack(mainCtx, airstackEndpoint, airstackKey)
```

Comment:
Please, only if API is provided. We want to be able to run the backend without Airstack support.